### PR TITLE
Make all indentation consistent (2)

### PR DESCRIPTION
**Pull request by Josh-Cena**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/webassembly/global/global/index.md [hunk 1]
```diff
@@ -67,11 +67,11 @@ const global = new WebAssembly.Global({value:'i32', mutable:true}, 0);
 
 WebAssembly.instantiateStreaming(fetch('global.wasm'), { js: { global } })
 .then(({instance}) => {
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/webassembly/global/global/index.md:69]: Shouldn't this be indented?

File: files/en-us/web/javascript/reference/global_objects/webassembly/global/index.md [hunk 2]
```diff
@@ -60,11 +60,11 @@ const global = new WebAssembly.Global({value:'i32', mutable:true}, 0);
 
 WebAssembly.instantiateStreaming(fetch('global.wasm'), { js: { global } })
 .then(({instance}) => {
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/webassembly/global/index.md:62]: Shouldn't this be indented?

File: files/en-us/web/javascript/reference/global_objects/webassembly/module/module/index.md [hunk 3]
```diff
@@ -68,7 +68,7 @@ fetch('simple.wasm').then((response) =>
   let mod = createWasmModule(bytes);
   WebAssembly.instantiate(mod, importObject)
   .then((result) =>
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/webassembly/module/module/index.md:70]: Shouldn't this be indented?

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/javascript/reference/global_objects/webassembly/global/index.md (hunk 2 above)
Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/global_objects/webassembly/global/index.md:62]: Ugh, indentation here is overall... gross, so I didn't touch it.

Comment (teoli2003) [reply files/en-us/web/javascript/reference/global_objects/webassembly/global/index.md:62]: I'm fine with leaving this for the moment. I will merge this then.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**